### ISSUE TITLE
[Bug]: Importing CSV files with a over 62175 rows of data fails

### ISSUE THREAD

**Issue by AndyDingo** [sqlitebrowser/sqlitebrowser#3652]
### What did you do?

Import > Table from CSV file >Browse for the 8MB csv file that has about 160 thousand rows > select it and then hit 'ok'

### What did you expect to see?

The data to be imported, even if it had been imported in smaller chunks, and taken longer.

### What did you see instead?

A dialog box with "Error importing data from record number 62175"

### DB4S Version

3.12.2

### What OS are you seeing the problem on?

Windows

### OS version

Windows 10 Pro 22H2 OS Build 19045.4412

### Relevant log output

```shell
SAVEPOINT "db4s_csvimport_17176530354639454";
SAVEPOINT "RESTOREPOINT";
CREATE TABLE "animal_species_flavor_text" (
	"species_id"	INTEGER,
	"version_id"	INTEGER,
	"language_id"	INTEGER,
	"flavor_text"	TEXT
);
PRAGMA database_list;
SELECT type,name,sql,tbl_name FROM "main".sqlite_master;
PRAGMA "main".TABLE_INFO("v_generation");
SELECT type,name,sql,tbl_name FROM sqlite_temp_master;
ROLLBACK TO SAVEPOINT "db4s_csvimport_17176530354639454";
```

### Prevention against duplicate issues

- [X] I have searched for similar issues

**Comment by justinclift** [comment]
Hmmm, that shouldn't happen.

As a quick test, maybe check if the appropriate Windows install from here has the same problem?

https://github.com/sqlitebrowser/sqlitebrowser/releases/tag/continuous

That's our latest development snapshot, with lots of bug fixes since the 3.12.2 release you're using.  These snapshots are pretty stable too, as we're close to a new release. :smile:

**Comment by AndyDingo** [comment]
Same issue on a more recent build (though with a far more helpful error message)
Though I managed to solve the issue by importing into excel, as recommended by [i-s-o], removing excess line characters, then exporting again.
